ansible/inventory/env/group_vars/all.yml

```diff
@@ -696,3 +696,7 @@ sunbird_portal_updateLoginTimeEnabled: false
 #sunbird_offline_azure_storage_account: "" #added this var for adopter usecase
 offline_installer_container_name: "" #added this var for adopter usecase
 cloud_storage_url: "{{ sunbird_public_storage_account_name }}.blob.core.windows.net"
+
+# Search-service
+search_index_host: "{{ groups['composite-search-cluster'][0] }}:9200"
```

Comment:
What if first host is unreachable in cluster? Shouldn't it take entire cluster comma separated IP.

Reply:
done